apps/condo/domains/user/schema/_oidc.test.js

```diff
@@ -208,6 +209,155 @@ describe('OIDC', () => {
                 expect(error).toHaveProperty(['request', 'res', 'statusCode'], 500)
             })
         })
+        describe('OIDCClient with "canAuthorizeSuperUsers: false" must not be used for authorizing super users',  () => {
+            test('canAuthorizeSuperUsers: false', async () => {
+                const uri = 'https://jwt.io/'
+                const clientId = getRandomString()
+                const clientSecret = getRandomString()
+                const admin = await makeLoggedInAdminClient()
+
+                await createTestOidcClient(admin, {
+                    payload: {
+                    // application_type, client_id, client_name, client_secret, client_uri, contacts, default_acr_values, default_max_age, grant_types, id_token_signed_response_alg, initiate_login_uri, jwks, jwks_uri, logo_uri, policy_uri, post_logout_redirect_uris, redirect_uris, require_auth_time, response_types, scope, sector_identifier_uri, subject_type, token_endpoint_auth_method, tos_uri, userinfo_signed_response_alg
+                        client_id: clientId,
+                        client_secret: clientSecret,
+                        redirect_uris: [uri], // using uri as redirect_uri to show the ID Token contents
+                        response_types: ['code id_token', 'code', 'id_token'],
+                        grant_types: ['implicit', 'authorization_code', 'refresh_token'], // 'implicit', 'authorization_code', 'refresh_token', or 'urn:ietf:params:oauth:grant-type:device_code'
+                        token_endpoint_auth_method: 'client_secret_basic',
+                    },
+                })
+
+                const c = await makeClientWithSupportUser()
```

Comment:
variables with such names from one symbol are difficult to read(

Reply:
Yeah, but I've just copied entire test from below. Changed client and expectations

Reply:
fixed naming to client